### ISSUE TITLE
Quantum Algorithms

### ISSUE THREAD

**Issue by AbhijeetGhosh** [OpenGenus/cosmos#4116]
**This is a(n):**

- [X] New algorithm
- [ ] Update to an existing algorithm
- [ ] Error
- [ ] Proposal to the Repository

**Details:**

This repository is missing all algorithms used for quantum computing and being named Cosmos it should contain every type of algorithms let alone skipping 'quantum algorithms'.

**Comment by pranaychatterjee** [comment]
Please assign this to me.

**Comment by UTx10101** [comment]
I want to work on this issue.
@AdiChat I need a confirmation of a Senior or Maintainer to know if Quantum Algorithms are in scope of Cosmos or not.

**Comment by UTx10101** [comment]
@AbhijeetGhosh Two world famous Quantum Algorithms have been added to Cosmos I think now the name Cosmos is best suited for this repository and you may close this issue. If you have suggestions for other algorithms which you think are important and must be added please feel free to share.

**Comment by AbhijeetGhosh** [comment]
> 
> 
> @AbhijeetGhosh Two world famous Quantum Algorithms have been added to Cosmos I think now the name Cosmos is best suited for this repository and you may close this issue. If you have suggestions for other algorithms which you think are important and must be added please feel free to share.

Yes I will, I expected a few other algorithms as well. I will close it in a couple of weeks
